src/comparisons/events/click/ie9.js

```diff
@@ -1 +1 @@
-el.addEventListener('click', function () {}, false);
+el.addEventListener('click', function () {});
```

Comment:
Any reason for dropping the capture arg here?

Reply:
[According to mdn](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener), `useCapture` is false by default.